src/screens/SettingsScreen.tsx

```diff
@@ -0,0 +1,277 @@
+                                    (systemTheme && !blackTheme) || systemTheme ? colors.LIGHT_ORANGE : colors.WHITE,
+                            },
+                        ]}
+                    >
+                        <TouchableOpacity
+                            onPress={() => {
+                                changeTheme(true, false);
+                            }}
+                        >
+                            <Text style={styles.systemHead}>{SettingScreenTheme.SYSTEM}</Text>
+                            <Text style={styles.systemComment}>{SettingScreenTheme.SYSTEM_COMMENT}</Text>
+                        </TouchableOpacity>
+                    </View>
+                    <View style={styles.buttonsThemeContainer}>
+                        <View
+                            style={[
+                                styles.buttonsTheme,
+                                { borderColor: !systemTheme && blackTheme ? colors.LIGHT_ORANGE : colors.WHITE },
+                            ]}
+                        >
+                            <TouchableOpacity
+                                onPress={() => {
+                                    changeTheme(false, true);
+                                }}
+                            >
+                                <Image style={styles.iconMoon} source={require('@assets/icons/icon-moon.png')} />
+                                <Text style={styles.textTheme}>{SettingScreenTheme.BLACK}</Text>
+                            </TouchableOpacity>
+                        </View>
+                        <View
+                            style={[
+                                styles.buttonsTheme,
+                                { borderColor: !systemTheme && !blackTheme ? colors.LIGHT_ORANGE : colors.WHITE },
+                            ]}
+                        >
+                            <TouchableOpacity
+                                onPress={() => {
+                                    changeTheme(false, false);
+                                }}
+                            >
+                                <Image style={styles.iconSun} source={require('@assets/icons/icon-sun.png')} />
+                                <Text style={styles.textTheme}>{SettingScreenTheme.LIGHT}</Text>
+                            </TouchableOpacity>
+                        </View>
+                    </View>
+                </View>
+            </View>
+        </View>
+    );
+};
+
+const styles = StyleSheet.create({
+    wrapperScreen: {
+        backgroundColor: colors.EXTRA_LIGHT_GRAY,
+        width: '100%',
+        height: '100%',
+    },
+    headScreen: {
+        width: '100%',
+        marginTop: 40,
+        marginBottom: 44,
+        flexDirection: 'row',
+        alignItems: 'center',
+        columnGap: 80,
+    },
+    titleOfScreen: {
+        fontFamily: 'ExpandedBold',
+        fontSize: 20,
+    },
+    iconBack: {
+        marginLeft: 40,
```

Comment:
As I remember it has the same offset as a lot of elements in the design (main horizontal offset)